core/trino-main/src/main/java/io/trino/metadata/SystemSecurityMetadata.java

```diff
@@ -161,12 +167,18 @@ default void validateEntityKindAndPrivileges(Session session, String entityKind,
 
     /**
      * Set the owner of the specified schema
+     *
+     * @deprecated {Use {@link #setEntityOwner(Session, String, List, TrinoPrincipal)}
      */
+    @Deprecated
     void setSchemaOwner(Session session, CatalogSchemaName schema, TrinoPrincipal principal);
```

Comment:
Sinces this is in the main module, we can remove these methods instead of deprecating them.  The only place we would leave the existing methods is in the trino-spi module.

Reply:
Removed.